sdk/search/search-documents/.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-  "cSpell.words": ["hnsw", "openai"]
+  "cSpell.words": ["hnsw", "openai", "Vectorizer"]
```

Comment:
Nit: can we move this to https://github.com/maorleger/azure-sdk-for-js/blob/main/.vscode/cspell.json and keep all the cspell in the same place?